index.html

```diff
@@ -0,0 +1,171 @@
+            </div>
+        </div>
+        <button class="subscribe button" type="button">Buy Halloween ed. Tickets</button>
+        <a href="#" class="buy-link">Buy Halloween ed. Tickets</a><br>
+    </section>
+    <section class="guest-appearences">
+        <div class="guests-header">
+            <h3>Guest Appearences</h3><hr>
+        </div>
+        <div class="guest-list">
+        <div class="guest-1 guesty">
+            <div class="image-guest" ><img id="guest1" src="#" alt="first-speaker"></div>
+            <div class="guest-details">
+                <h4 id="guest-name1" class= "detal">1</h4>
+                <h5 id="guest-prof1" class= "detal guest-prof">1</h5><hr class="line">
+                <p id="guest-career1" class= "detal"></p>
+            </div>    
+        </div>
+        <div class="guest-2 guesty">
+            <div class="image-guest"><img  id="guest2" src="#" alt="first-speaker"></div>
+            <div class="guest-details">
+                <h4 id="guest-name2" class= "detal">2</h4>
+                <h5 id="guest-prof2" class= "detal guest-prof">2</h5><hr class="line">
+                <p id="guest-career2" class= "detal"></p>
+            </div>    
+        </div>
+        <div class="guest-3 guesty">
+            <div class="image-guest"><img  id="guest3" src="#" alt="first-speaker"></div>
+            <div class="guest-details">
+                <h4 id="guest-name3" class= "detal">3</h4>
+                <h5 id="guest-prof3" class= "detal guest-prof">3</h5><hr class="line">
+                <p id="guest-career3" class= "detal"></p>
+            </div>    
+        </div>
+        <div class="guest-4 guesty">
+            <div class="image-guest"><img  id="guest4" src="#" alt="first-speaker"></div>
+            <div class="guest-details">
+                <h4 id="guest-name4" class= "detal">4</h4>
+                <h5 id="guest-prof4" class= "detal guest-prof">4</h5><hr class="line">
+                <p id="guest-career4" class= "detal"></p>
+            </div>    
+        </div>
+        <div class="guest-5 guesty">
+            <div class="image-guest"><img id="guest5" src="#" alt="first-speaker"></div>
+            <div class="guest-details">
+                <h4 id="guest-name5" class= "detal">5</h4>
+                <h5 id="guest-prof5" class= "detal guest-prof">5</h5><hr class="line">
+                <p id="guest-career5" class= "detal"></p>
+            </div>    
+        </div>
+        <div class="guest-6 guesty">
+            <div class="image-guest"><img id="guest6" src="#" alt="first-speaker"></div>
+            <div class="guest-details">
+                <h4 id="guest-name6" class= "detal">6</h4>
+                <h5 id="guest-prof6" class= "detal guest-prof">6</h5><hr class="line">
+                <p id="guest-career6" class= "detal"></p>
+            </div>    
+        </div>
+        </div>
```

Comment:
- [ ] Since this is a guest list, I think we could use the more semantic tags `ul` and `li`.